packages/js/src/executors/tsc/schema.json

```diff
@@ -61,6 +61,31 @@
       "description": "When `updateBuildableProjectDepsInPackageJson` is `true`, this adds dependencies to either `peerDependencies` or `dependencies`.",
       "enum": ["dependencies", "peerDependencies"],
       "default": "peerDependencies"
+    },
+    "external": {
+      "oneOf": [
+        {
+          "type": "string",
+          "description": "Mark which other projects to be external to the project. Eg: projects need to be published and installed separately",
```

Comment:
This description and `default: all` should go up a level to preserve existing behavior.

Minor nitpick: descriptions should end with a period `.`.